src/modules/Library/features/Applet/Item/Item.utils.tsx

```diff
@@ -0,0 +1,39 @@
+import uniqueId from 'lodash.uniqueid';
+
+import { variables, StyledBodyLarge } from 'shared/styles';
+import i18n from 'i18n';
+
+import { StyledItemContentRow, StyledItemSvg, StyledItemImage } from './Item.styles';
+import { Item, ItemResponseType } from '../../AppletsCatalog/AppletsCatalog.types';
+import { itemResponseTypes } from '../../AppletsCatalog/AppletsCatalog.conts';
+
+export const renderItemContent = (item: Item) => {
+  const { t } = i18n;
+
+  switch (item.responseType) {
+    case ItemResponseType.SingleSelection:
+    case ItemResponseType.MultipleSelection:
+      return (
+        <>
+          {item.options?.map(({ title, image }) => (
+            <StyledItemContentRow key={uniqueId()}>
+              <StyledItemSvg>{itemResponseTypes[item.responseType].icon}</StyledItemSvg>
+              {image && <StyledItemImage src={image} alt="Option image" />}
```

Comment:
Should we use translation for `alt="Option image"`?

Reply:
hm, as I know, we don't translate alt